pages.es/linux/debuginfod-find.md

```diff
@@ -0,0 +1,8 @@
+# debuginfod-find
+
+> Solicita datos relacionados con debuginfo.
+> Más información: <https://manned.org/debuginfod-find>.
+
+- Solicita datos basados en `build_id`:
+
+`debuginfod-find -vv debuginfo {{build_id}}`
```

Comment:
```suggestion
`debuginfod-find -vv debuginfo {{identificador_de_build}}`
```

Reply:
As it is a quite common pattern, I think we should document this in the style guide so that future contributors know this

Reply:
I am sorry, I did not understand what is the common pattern. The use of prepositions in placeholders or writing identificador instead of `id`?

Reply:
Yeah, sorry, I didn't explain the pattern. In Spanish pages, "id" must be translated to "identificador"